src/test/resources/cz/startnet/utils/pgdiff/add_constraint_diff.sql

```diff
@@ -1,3 +1,3 @@
 
 ALTER TABLE testtable
-	ADD CONSTRAINT IF NOT EXISTS field4check CHECK (((field4 > (-5.0)::double precision) AND (field4 < (5.0)::double precision)));
```

Comment:
same here, what's wrong with the "if not exists" ?

Reply:
The constraint related code was removed. Only sequence code in the PR now